src/encoded/tests/data/inserts/target.json

```diff
@@ -1,12 +1,6 @@
 [
     {
-        "dbxref": [
-            "ENSEMBL:ENSG00000101126",
-            "GeneID:23394",
-            "HGNC:ADNP1",
-            "UniProtKB:Q9H2P0"
-        ],
-        "gene_name": "ADNP",
+        "genes": ["23394"],
```

Comment:
So the HGNC numerical ID is a valid alias -> pointer to the gene... that might be a little confusing.

Reply:
This is NCBI Entrez ID which is the key for gene object. It's confusing without namespace. Do you think add some explanation to the schema will help?